### DISCUSSION TITLE
Element Absolute position

### DISCUSSION THREAD

**Discussion by ghost** [QuestPDF/QuestPDF#836] [Q&A] [answered]
Hi,
I was trying to generate a PDF composed of multiple A4 pages with my .NET 6 application and I came across a small issue.

Considering I need to create a text element that will overlap on a image, how can I implement something like that?
I've tried using the Translate function, Column and Row containers but I'm not getting the expected result.
Is there some way to generate an element and just give it a specific size (width and height) and eventually the X,Y point where to draw it? 

Here's some code I was using:
container.Page(page =>
{
    page.Size(PageSizes.A4.Landscape());
    page.Header()
        .Image(@"D:\header_pdf.png");

    page.Footer()
    .Row(x =>
    {
        x.RelativeItem().Image(@"D:\footer_pdf.png");
        x.ConstantItem(50).Text("ID: 000000/513");
    });
       
});

Thanks in advance

**Comment by HightowerCZ** [comment, marked as answer]
Have you tried layers?

**Comment by twofingerrightclick** [comment]
You probably want to make the "absolute" item unconstrained : Unconstrained(), and then use Translate() to move it to the position you want.
https://www.questpdf.com/api-reference/unconstrained.html#unconstrained

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

As the community members suggested, the [Layers](https://www.questpdf.com/api-reference/layers.html) element is the recommended way to overlay content such as text on top of an image.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
